.github/dependabot.yml

```diff
@@ -6,3 +6,8 @@ updates:
     interval: daily
     time: "10:00"
   open-pull-requests-limit: 10
+  ignore:
```

Comment:
The [docs](https://dependabot.com/docs/config-file/#ignored_updates) you linked to earlier suggest this should be `ignored_updates`? Did this change when Dependabot was integrated into GitHub?

Reply:
> Did this change when Dependabot was integrated into GitHub?

Exactly. I used the "Dependabot Preview" (legacy non-native version) web interface to PR the initial dependabot.yml (#1258), which also did the migration to GitHub-native, which seems to have a slightly different config syntax: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates#disabling-dependabot-version-updates